defs.h

```diff
@@ -68,6 +68,9 @@
 
 #define MAXPATHNODES			300
 
+// Careful! Changing this variable causes saves with a different one to crash! Default: 25

```

Comment:
shouldn't it just cause them to be incompatible?

Reply:
well I just added that line because I don't think we have any other defines that change stuff that gets saved atm

Reply:
Maxprortals does

Reply:
then you can just remove this line :P

Reply:
```suggestion
```